src/Extractor/HttpClient.php

```diff
@@ -543,7 +527,7 @@ private function getUglyURL(string $url, string $html)
     {
         $found = false;
         foreach ($this->config['ajax_triggers'] as $string) {
-            if (stripos($html, $string)) {
+            if (stripos($html, (string) $string)) {
```

Comment:
We should either use value object with typed properties or otherwise validate the config when passed to Graby. Then it would not add pointless casts like this.

Reply:
I would like to implement https://symfony.com/doc/current/components/options_resolver.html#type-validation